README.md

```diff
@@ -140,12 +118,12 @@ The uploaded image is assigned a randomly generated public Id. As soon as `onSuc
     http://res.cloudinary.com/demo/image/upload/abcfrmo8zul1mafopawefg.jpg
 
 You can also specify your own public ID:
-    
+
     String requestId = MediaManager.get().upload(uri).unsigned("sample_preset").option("public_id", "sample_remote").dispatch();
 
 Using `RequestUploadPolicy`, an upload request can be configured to run under specific circumstance, or within a chosen time window:
 
-The following examples uploads local Uri resource, configured to run immediately (the default), with a maximum of 7 retries, and only on an unmetered network (e.g. wifi):
+The following examples uploads local Uri resource, configured to run immediately (the default), with a maximum of 7 retries, and only on an un-metered network (e.g. wifi):
```

Comment:
In general, words with 'un' should not be hyphenated unless the word could have another meaning without the hyphen.  And from everything I see on Google, this term is always used without a hyphen, so we should revert this one.

Reply:
Done, thank you